packages/api/tests/lib/test-granules.js

```diff
@@ -91,3 +282,221 @@ test('getGranuleProductVolume() returns correct product volume', (t) => {
     0
   );
 });
+
+test.serial('deleteGranuleAndFiles() removes a granule from PG and Dynamo', async (t) => {
+  const { newPgGranule, newDynamoGranule } = await createGranuleAndFiles(
+    t.context.knex,
+    t.context.testPgCollection.cumulus_id,
+    false
+  );
+
+  await deleteGranuleAndFiles({
+    knex: t.context.knex,
+    dynamoGranule: newDynamoGranule,
+    pgGranule: newPgGranule,
+    granuleModelClient: granuleModel,
+  });
+
+  // Check Dynamo and RDS. The granule should have been removed from both.
+  await t.throwsAsync(
+    granuleModel.get({ granuleId: newDynamoGranule.granuleId }),
+    { instanceOf: RecordDoesNotExist }
+  );
+
+  await t.throwsAsync(
+    granulePgModel.get(t.context.knex, { granule_id: newPgGranule.granule_id }),
+    { instanceOf: RecordDoesNotExist }
```

Comment:
Use `exists` model methods

Reply:
Sorry, I should have made a pass through all of the tests based on your previous comment. Got these and found 2 more:

d7230248ee1b906511c141188c56e31173326976
b1f5358e81d5cccc4117c980f75325785427c36c

Reply:
9th time's the charm: https://github.com/nasa/cumulus/commit/c975861f3a0b9ed3b2fdd916add209ae20952533